ProffieOS.ino

```diff
@@ -705,6 +706,10 @@ int prop_GetBulletCount() {
 }
 #endif
 
+int prop_GetBlasterMode() {
+    return prop.GetBlasterMode();
```

Comment:
This won't compile if the prop doesn't have a GetBlasterMode function.
To fix that, we should move this function from ProffieOS.ino to props/blaster.h

Reply:
I tried that, but there's an issue with the order of things. It seems 'prop' variable isn't available to blaster.h

Reply:
Right, good point.
How about we add a #define PROP_HAS_GETBLASTERMODE in blaster.h then, and then we'll use an ifdef here.

Reply:
I'll work on that. 
Also, (off, but similar topic) if a non-blaster prop is included in the user config, 
but also #define BLASTER_SHOTS_UNTIL_EMPTY is active, an error:
"class SaberBCButtons' has no member named 'GetBulletCount'"

Now, why would someone have a bullet count define for a saber config is the question, but maybe some other error should show just in case they do? like "Blaster defines are not supported in saber configs" or something?

Reply:
> add a #define PROP_HAS_GETBLASTERMODE in blaster.h

ok done.

Reply:
> Now, why would someone have a bullet count define for a saber config is the question,

I'm more worried about the opposite.
Not all blasters have limited number of shots.